Exercise04/index.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
+const fs = require("fs");
+const _ = require('lodash');
+
+const PORT = 3000;
+const ERROR_ID = -1;
+const taskPath = './tasks.json';
+
+let app = express();
+
+app.get('/', (req, res) => { 
+    res.send("Welcome to Idan's TODO list server");
+});
+
+// /tasks - show all tasks
+app.get('/tasks', (req, res) => { 
+     fs.readFile(taskPath, 'utf8', (err, taskFile) => {
```

Comment:
Think about moving all of this to a function since you'll be using it in some more places.